boardStupid.php

```diff
@@ -0,0 +1,24 @@
+<?php
+session_start();
+
+$db = new PDO('mysql:host=db; dbname=boardStupid', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT *  FROM `boardStupid`;');
+$query->execute();
+$result = $query->fetchAll();
+
+require_once 'game_function.php'
+
+?>
+
+  
+
+<?php
+
+echo '<h1>boardStupid</h1>';
+
+echo printStats($result);
+
+'</body>';
+'</html>';
```

Comment:
You don't seem to have the corresponding opening tags, as well as the other HTML bits and bobs like the head section (would be worth linking to the normalize.css file in there)